index.js

```diff
@@ -3,14 +3,13 @@
 var fs = require('fs');
 var path = require('path');
 var log = require('gulplog');
+var fancyLog = require('fancy-log');
```

Comment:
I really don't like using fancy-log directly, it means that someone using `--silent` will still see the messages.  Looking into re-binding the logger after we parse the config.